Assignment/solution_student/css/style.css

```diff
@@ -1,27 +1,239 @@
-/* BASE STYLES
+/* UTILITY
 /*******************************************/
-body {
-  color: black;
-  font-size: 16px;
-  background: white;
+
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+html {
+  box-sizing: border-box;
+}
+.clearfix {
+  *zoom: 1;
+}
+.clearfix:before,
+.clearfix:after {
+    content: " ";
+    display: table;
+}
+.clearfix:after {
+  clear: both;
 }
 
+/*i started off with ALL intentions of utilizing the categories 
+outlined in the CSS template. that is SO hard to do when you are
+constantly trying to figure everything out. i didn't do it this 
+time, but if i have time i'll go back in and sort. i'm sure 
+duplication of code is annoying to review.*/
 /******************************************
-/* LAYOUT
+/* BASE STYLES
 /*******************************************/
+body {
+	color: black;
+	margin: 0;
+	font-size: 16px;
+	font-family: 'open sans', sans-serif;
+	width: 100%;
+}
 
-/******************************************
-/* PAGES
-/*******************************************/
+/****************************************/
+/*it looks like there is an opacity piece on top of my image, 
+but i can't figure it out! why is it so hazy?*/
+header {
+	background-image: url("../images/header_bg.jpg");
+	background-repeat: no-repeat;
+	height: 400px;
+	width: 100%;
+	background-size: cover;
+}
+
+footer {
+	background: #121212;
+	background-size: cover;
+	width: 100%;
+	height: 250px;
+	padding-top: 10px;
+	text-align: center;
+	opacity: 0.9;
+
+}
+
+/*the google font weights look very thick (compared to the PNG)
+i tweaked, but i read the doc! trust you, me. i try to follow
+directions*/
+h1, h2 { 
+	font-size: 24px;
+	color: #f0efef;
+}
+
+h1 {
+	font-weight: 700;	
+	margin-left: 50px;
+	margin-top: 0px;
+	margin-bottom: 0px;
+	padding-top: 20px;
+	padding-left: 50px;
+}
+
+h2 {
+	font-weight: 500;
+	margin-left: 300px;
+	margin-bottom: 0px;
+	margin-top: 0px;
+	padding-top: 50px;
+	padding-bottom: 80px;
+}
+
+h3 {
+	font-size: 20px;
+	font-weight: 600;
+	color: #033048;
+	margin-left: 50px;
+	margin-top: 0px;
+	margin-bottom: 0px;
+	padding-top: 30px;
+	padding-bottom: 0px;
+}
+
+/*hmmm, i am now questioning where to use margin and where to
+use padding. rule of thumb?*/
+h4 { 
+	color: #f9e42e;
+	padding-top: 20px;
+	font-size: 24px;
+	padding-bottom: 25px;
+	margin: 0px;
+}
+
+h5 {
+	font-size: 9px;
+	font-style: bold;
+	color: white;
+	padding-top: 10px;
+}
+
+/*i'm bummed i couldn't really figure out this spacing. i 
+changed margins instead of using <br> but i don't understand
+how margins work with fixed elements*/
+p {
+	color: #606161;
+	font-size: 13px;
+	font-weight: lighter;
+	margin-left: 50px;
+	margin-right: 650px;
+}
+
+ul {
+	color: #606161;
+	font-size: 13px;
+	font-weight: lighter;
+	margin-left: 50px;
+	margin-right: 700px;
+	padding-top: 20px;
+	padding-bottom: 30px;
+	margin-bottom: 0px;
+}
+
+/*how do i get this button longer? i tried to change the width
+(see below) and nothing! */
```

Comment:
The reason it ignored your `width` declaration is because an `a` tag is an inline element, and remember, inline elements can't have explicit widths, so you have to change it to `display: inline-block;` so it will stay centered like an inline element, but give you the ability to change its width.